deploy/examples/object-multisite-test.yaml

```diff
@@ -35,6 +35,8 @@ spec:
       requireSafeReplicaSize: false
     parameters:
       compression_mode: none
+  # customEndpoints:
```

Comment:
In the examples let's add a comment that it's recommended to set this value for their ingress endpoint